app/api/chat/route.ts

```
@@ -0,0 +1,479 @@
+        break
+      
+      case 'GOOGLE':
+        response = await callGemini(apiKey, message, conversationHistory)
+        // Estimate tokens for Gemini (it doesn't provide exact counts)
+        promptTokens = Math.ceil(message.length / 4)
+        completionTokens = Math.ceil(response.content.length / 4)
+        break
+      
+      case 'PERPLEXITY':
+        // Perplexity uses 'sonar-pro' but their API expects 'sonar'
+        const perplexityModel = model === 'sonar-pro' ? 'sonar' : model
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)